recipe/meta.yaml

```diff
@@ -1,35 +1,48 @@
-{% set version = "1.10" %}
-{% set sha256 = "bcbe8ada8c435a253b1fa438f505dd311832a88f52316fd7d30342b106a4242c" %}
-{% set runc_version = "0.1.0" %}
-{% set runc_sha256 = "78ae42905c0f26fcfa9353e48e1a001b5b5c3c94b33e2ceacc6bc84c23c61a88" %}
+{% set goname = "github.com/tianon/gosu" %}
+{% set version = "1.11" %}
+
+{% set name = goname.split('/')[-1] %}
+{% set pkg_src = ('src/'+goname).replace("/",os.sep) %}
```

Comment:
Also, the `os.sep` call in L5 is what causing the linter to fail. I already submitted a PR to conda-forge/conda-smithy#980 which was merged but is pending a release.

Reply:
in the meantime to get the error out I replaced it with an inline if statement.